crates/rooch/src/commands/account/commands/nullify.rs

```diff
@@ -26,59 +26,55 @@ pub struct NullifyCommand {
     address: String,
     #[clap(flatten)]
     pub context_options: WalletContextOptions,
-    /// Command line input of coin ids
-    #[clap(short = 'c', long = "coin-id", arg_enum)]
-    pub coin_id: CoinID,
+    /// Command line input of multichain ids
+    #[clap(short = 'i', long = "multichain-id")]
+    pub multichain_id: RoochMultiChainID,
```

Comment:
I think we can remove the multichain id here. It should be the validator sheme id, but not every validator scheme have auth key. So, just remove and keep simple.

Reply:
Removed.